de-lib/src/main/java/org/iplantc/de/notifications/client/utils/NotificationUtil.java

```diff
@@ -66,7 +68,10 @@
     @Inject AsyncProviderWrapper<JoinTeamRequestDialog> joinRequestDlgProvider;
     @Inject AsyncProviderWrapper<DenyJoinRequestDetailsDialog> denyDetailsDlgProvider;
     @Inject DesktopView.Presenter.DesktopPresenterAppearance appearance;
-
+    //@Inject
+    //NotificationView.NotificationViewAppearance notificationViewAppearance;  This doesn't seem to work
+    NotificationView.NotificationViewAppearance notificationViewAppearance =
+            GWT.create(NotificationView.NotificationViewAppearance.class);
     @Inject
     public NotificationUtil() {
```

Comment:
That's weird. What if instead of injecting it with the annotation, you inject it through the constructor?

Reply:
I tried that as well. But that require a refactor to `NotificationCallbackConverter` since that class is not injected right now.

Reply:
Ah ok, not really something worth worrying about in this PR 👍